lib/konbata/configuration.rb

```diff
@@ -2,13 +2,23 @@
 
 module Konbata
   class Configuration
+    attr_accessor :canvas_url
+    attr_accessor :canvas_token
+    attr_accessor :account_id
+    attr_accessor :request_timeout
```

Comment:
Is there a reason we're using `attr_accessor` for these four pieces of data? I think it'd probably be better if we used `attr_reader` instead since there shouldn't ever be a reason for another object to modify them. They're just being read in from the config file.